### DISCUSSION TITLE
All badges

### DISCUSSION THREAD

**Discussion by AntoVerga** [drknzz/GitHub-Achievements#258] [Q&A] [answered]
I can have all the badges or it's a limited number that I can have on my profile?

**Comment by androsM2** [comment]
> I can have all the badges or it's a limited number that I can have on my profile?

you can

**Comment by the1Riddle** [comment, marked as answer]
you can absolutely have all the badges @AntoVerga 

here is an example but not completely all as in the readme for to get some you need to be awarded an example i have seen is the Open Sourcerer if not wrong. 
![image](https://github.com/drknzz/GitHub-Achievements/assets/125451537/6747e28d-740c-4fd4-9afe-3f3a5523b3e5)

**can you please mark me as answer**...

**Reply by Ishmael-quant** [reply]
this looks amazing.

**Reply by Ishmael-quant** [reply]
but currently you can only get the nine of them as `Mars 2020 Contributor` and `Arctic Code Vault Contributor` are not being issued as badges.

**Reply by the1Riddle** [reply]
yeah, that's true
